src/components/Form.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react'
+import Button from './Button.js'
+import Preview from './forms/Preview'
+import Section from './forms/Section'
+import "../styles/form.css"
+
+export default class Form extends Component {
+
+  constructor( props ) {
+    super( props )
+    this.state = {}
+    this.setFieldState = this.setFieldState.bind( this )
```

Comment:
bind inline, instead of in constructor (this would allow us to remove this constructor, saving 5 lines of code)